pipeline_steps/pubcloud.groovy

```diff
@@ -237,8 +237,8 @@ def runonpubcloud(Map args=[:], Closure body){
 }
 
 def uploadToSwift(Map args){
-  if(fileExists("${WORKSPACE}/artifacts")){
-    print("WORKSPACE/artifacts directory found, Preparing to upload artifacts.")
+  if (fileExists(args.path)) {
+    print("Directory ${args.path} found. Uploading contents.")
```

Comment:
Whats args.path for, couldn't args.artifact_dir be used instead?

Reply:
I was trying to move towards ``uploadToSwift`` be something more generically usable for more than just the artifact upload purpose. I will admit to thinking that perhaps the change of var name makes it just a little harder to follow in the current way this is used though. I can change it up if you'd like.

Reply:
@odyssey4me My question isn't so much the name, but rather are both required?

Reply:
@hughsaunders ``args.artifact_dir`` is not a valid parameter for the function, unless I rename ``args.path``.